### PR TITLE
Remove hardcoded text

### PR DESCRIPTION
**Pull request by jarrodmillman**

### CODE REVIEW

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -1,12 +1,15 @@
+{{- $title          := .Site.Params.title }}
 {{- $footer      := .Site.Params.footer }}
 {{- $logo        := index $footer "logo" }}
 {{- $quickLinks  := index $footer "quicklinks" }}
 {{- $socialMedia := index $footer "socialmedia" }}
+{{- $navbarLogo     := .Site.Params.navbarlogo }}
```
Comment (stefanv) [inline layouts/partials/footer.html:6]: Shall we keep the capitalization constant throughout?

File: static/images/README.md [hunk 2]
```diff
@@ -1,4 +1,19 @@
-inkscape -w 16 -h 16 -o 16.png logos/scipy.svg
-inkscape -w 32 -h 32 -o 32.png logos/scipy.svg
-inkscape -w 48 -h 48 -o 48.png logos/scipy.svg
+Add logo with name `logo.svg`.
```
Comment (stefanv) [inline static/images/README.md:1]: Looks like the main logo has a fixed filename, but the footer logo is still customizable?  Perhaps both or none shoul dbe.

**Review by stefanv — COMMENTED** [review]
Looks good.  We may want to retain the camelCase for long parameter names in the configuration file?

File: layouts/partials/footer.html (hunk 1 above)
Comment (jarrodmillman) [reply layouts/partials/footer.html:6]: I will do that in a followup PR, but I would first like to test this with numpy.org.

File: static/images/README.md (hunk 2 above)
Comment (jarrodmillman) [reply static/images/README.md:1]: I think I got that one too, now.

**Pull request merged**